src/traits/sign.rs

```diff
@@ -4,9 +4,21 @@ use helium_api::{
     BlockchainTxnAddGatewayV1, BlockchainTxnAssertLocationV1, BlockchainTxnCreateHtlcV1,
     BlockchainTxnOuiV1, BlockchainTxnPaymentV1, BlockchainTxnPaymentV2, BlockchainTxnPriceOracleV1,
     BlockchainTxnRedeemHtlcV1, BlockchainTxnSecurityExchangeV1, BlockchainTxnTokenBurnV1,
-    BlockchainTxnVarsV1, Message,
+    BlockchainTxnTransferHotspotV1, BlockchainTxnVarsV1, Message,
 };
 
+#[derive(PartialEq)]
```

Comment:
How did this suddenly show up here as added lines?

Reply:
ach - mistake while rebasing.